app/controllers/support_controller.rb

```diff
@@ -30,7 +30,7 @@ def create
       redirect_to support_thanks_path
     else
       flash[:errors] = @support.errors
-      if params[:subject] == "problem"
+      if params[:support][:subject] == "[Problem]"
```

Comment:
very minor but you don't need double quotes here as you are not using string interpolation.

Reply:
No problem, Ive changed them to single quotes along with "problem" to match "[Problem]"